web/src/components/core/Sidebar.jsx

```diff
@@ -130,8 +130,9 @@ export default function Sidebar ({ children, isOpen, onClose = noop }) {
     <aside
       id="global-options"
       ref={asideRef}
-      className="wrapper sidebar"
       aria-label={_("Installer Options")}
+      data-type="agama/sidebar"
+      data-layout="agama/base"
```

Comment:
**A note about using `data-` attributes for styling**

---

I started using them for _exceptions_, as [suggested by CUBE](https://cube.fyi/exception.html#exception) methodology which I'd like to follow (WIP). But lately I found myself using them more to make styles easier to understand and, at some level, improve DX.

I'm aware that CSS _is_ mostly about classes selectors and they are likely to be more performant than data- attributes when used for styling. Though I haven't tested it myself, the potential performance penalty appears to be insignificant. See https://webdev.tips/styling-using-data-attributes#heading-performance-considerations, https://ecss.benfrain.com/appendix1.html, and https://gomakethings.com/how-performant-are-data-attributes-as-selectors/

Keeping a balance is the key here. Making use of them when it seems reasonable, and most importantly, trying not to abuse them. It's actually **my** goal to **use as few selectors as possible**, regardless of their type.

Of course, I can be wrong when taking a decision of adding or removing a selector. And that's why I'm adding this note, to let you know what's the main intention behind my _CSS movements_ and to encourage you to join me in the joy of exploring how to make the Agama CSS great and **minimal** at the same time :)

Also, I've to say that I'm not afraid to make mistakes here. They give me/us the opportunity to learn and improve. Furthermore, CSS **is easy to change and fix**.

BTW, when looking for evidences about data- selectors performance I found an article talking about [Class-less CSS architecture](https://jacobxperez.github.io/blog/post/css/class-less-architecture/)  :innocent: